ruby/cross_service_examples/item-tracker/app.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
```

Comment:
Missing Apache license.

Reply:
The majority of Ruby files do not have licensing because it appears elsewhere at a higher level. Do I need to be adding this to every single Ruby file?

Reply:
Yes, it's been my understanding we have to include this notice at the top of every code file.